app/components/primer/avatar_stack_component.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module Primer
+  # Use AvatarStack to stack multiple avatars together.
+  class AvatarStackComponent < Primer::Component
+    include ViewComponent::SlotableV2
+
+    ALIGN_DEFAULT = :left
+    ALIGN_OPTIONS = [ALIGN_DEFAULT, :right].freeze
+
+    # Required list of stacked avatars.
+    #
+    # @param kwargs [Hash] The same arguments as <%= link_to_component(Primer::AvatarComponent) %>.
+    renders_many :avatars, Primer::AvatarComponent
+
+    # @example auto|Default
+    #   <%= render(Primer::AvatarStackComponent.new) do |c| %>
+    #     <%= c.avatar(src: "http://placekitten.com/200/200", alt: "@kittenuser") %>
+    #     <%= c.avatar(src: "http://placekitten.com/200/200", alt: "@kittenuser") %>
+    #     <%= c.avatar(src: "http://placekitten.com/200/200", alt: "@kittenuser") %>
+    #   <% end  %>
+    #
+    # @example auto|Align right
+    #   <%= render(Primer::AvatarStackComponent.new(align: :right)) do |c| %>
+    #     <%= c.avatar(src: "http://placekitten.com/200/200", alt: "@kittenuser") %>
+    #     <%= c.avatar(src: "http://placekitten.com/200/200", alt: "@kittenuser") %>
+    #     <%= c.avatar(src: "http://placekitten.com/200/200", alt: "@kittenuser") %>
+    #   <% end  %>
+    #
+    # @example auto|With tooltip
+    #   <%= render(Primer::AvatarStackComponent.new(tooltipped: true, body_arguments: { label: 'This is a tooltip!' })) do |c| %>
+    #     <%= c.avatar(src: "http://placekitten.com/200/200", alt: "@kittenuser") %>
+    #     <%= c.avatar(src: "http://placekitten.com/200/200", alt: "@kittenuser") %>
+    #     <%= c.avatar(src: "http://placekitten.com/200/200", alt: "@kittenuser") %>
+    #   <% end  %>
+    #
+    # @param align [Symbol] <%= one_of(Primer::AvatarStackComponent::ALIGN_OPTIONS) %>
+    # @param tooltipped [Boolean] Whether to add a tooltip to the stack or not
+    # @param body_arguments [Hash] Parameters to add to the Body. If `tooltipped` is set, has the same arguments as <%= link_to_component(Primer::TooltipComponent) %>.
```

Comment:
Could we confirm the behavior is not winky if `tooltipped: true` is set but no body params are passed?

Reply:
The component will raise, because if `tooltipped == true`, `body_params` requires `label:`
I can add a test checking this behavior

Reply:
Raising seems unfortunate, in my opinion. I'd expect to end up with no tooltip at all. WDYT?

Reply:
I think it is ok to raise. If you call a `TooltipComponent` without `label:` it will also raise
That makes it easier for the user to know what is wrong instead of simply not rendering anything

Reply:
I was thinking more about the production use case where the tooltip value might be dynamically generated and there's some bug in the user code where it's missing. I'd not expect my page to blow up but I appreciate that passing the wrong args makes sense to blow up in dev. Does the library have a philosophy on that kind of question?

Reply:
I don't think we have a philosophy for that case. That's something we can discuss more!
Right now, if an user does not provide a **required** argument to a component, it will break

Reply:
Maybe that's a smell on this component API, that's why I initially wanted `body` to be a slot, but couldn't because this specific use is not supported by the framework